x/ccv/consumer/module.go

```diff
@@ -159,13 +159,14 @@ func (am AppModule) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) {
 		// the consumer chain is no longer safe
 
 		channelClosedMsg := fmt.Sprintf("CCV channel %q was closed - shutdown consumer chain since it is not secured anymore", channelID)
-		ctx.Logger().Error(channelClosedMsg)
+		am.keeper.Logger(ctx).Error(channelClosedMsg)
 		panic(channelClosedMsg)
 	}
 
-	blockHeight := uint64(ctx.BlockHeight())
+	blockHeight := uint64(ctx.BlockHeight()) + 1
```

Comment:
The next line is supposed to retrieve the VSC ID mapped to the **current** block height. This should be reverted to: 
```suggestion
	blockHeight := uint64(ctx.BlockHeight())
```